modin/pandas/test/test_io.py

```diff
@@ -1555,6 +1556,19 @@ def test_read_parquet_pandas_index(self, engine, filters):
 
         for col in pandas_df.columns:
             if col.startswith("idx"):
+                # Before this commit, first released in version 2023.1.0, fastparquet relied
+                # on pandas private APIs to handle Categorical indices.
+                # These private APIs broke in pandas 2.
+                # https://github.com/dask/fastparquet/commit/cf60ae0e9a9ca57afc7a8da98d8c0423db1c0c53
+                if (
+                    col == "idx_categorical"
+                    and engine == "fastparquet"
+                    and version.parse(fastparquet.__version__)
+                    < version.parse("2023.1.0")
+                    and version.parse(pandas.__version__) >= version.parse("2.0.0")
```

Comment:
I don't think we need this particular check, we aren't supporting pandas 1.x anymore, and probably won't in any foreseeable future.

Reply:
Makes sense, I'll remove this.